### DISCUSSION TITLE
Attributes and powers over 5

### DISCUSSION THREAD

**Discussion by Alepagano79** [JohanFalt/Foundry_WoD20#875] [Q&A] [answered]
Good morning,
is possible create attributes, abilities and powers over 5. I would create sheets of ancient vampires.
Thank you for reponse!

**Comment by JohanFalt** [comment]
Easiest is to just set their generation to what you want the ancient vampire to have.

Out of my head I believe the setting is under the sheet setting and then attributes.

**Comment by JohanFalt** [comment]
No Bio:

https://github.com/JohanFalt/Foundry_WoD20/wiki/Sheet:-Vampire-the-Masquerade#bio

**Comment by SuperSoldier240** [comment]
I'm also interested in using Powers over 5, specifically spheres for Archmages to use. Is there any current way to do that?

**Reply by JohanFalt** [reply]
No, but I'll add this as it is missing.

**Reply by SuperSoldier240** [reply]
Thank you!

**Comment by JohanFalt** [comment]
This exists in v4.1.

**Reply by JohanFalt** [reply]
But not mages yet, I'll add that.

**Comment by JohanFalt** [comment, marked as answer]
This works in v4.1.1 now.
